emily/handler/src/database/accessors.rs

```diff
@@ -208,7 +209,9 @@ pub async fn pull_and_update_deposit_with_retry(
             DepositUpdatePackage::try_from(&deposit_entry, update.clone())?;
         // Attempt to update the deposit.
         match update_deposit(context, &update_package).await {
-            Err(Error::VersionConflict) => {
+            Err(Error::VersionConflict(error)) => {
+                warn!(%error, "received an error when updating a deposit request");
```

Comment:
Seems the convention in emily has been to use "Uppercase first words".

Reply:
Found one more https://github.com/stacks-sbtc/sbtc/pull/1627/commits/4a903835b147cdb4fd117ec37d010fd983a0d0de